14-tidyr.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+layout: page
+title: R for reproducible scientific analysis
+subtitle: Dataframe manipulation with tidyr
+minutes: 60
+---
+
+```{r, include=FALSE}
+source("tools/chunk-options.R")
+opts_chunk$set(fig.path = "fig/12-plyr-")
+# Silently load in the data so the rest of the lesson works
+gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
+```
+
+> ## Learning objectives {.objectives}
+>
+> * To be able to convert dataframes between 'long' and 'wide' format with `tidyr`
+>
+
+Researchers often want to manipulate their data from the 'wide' to the 'long' format, or vice-versa. The 'long' format is where:
+
+ - each column is a variable
+ - each row is an observation
+
+In the 'long' format, you usually have 1 column for the observed variable and the other columns are ID variables. 
+
+For the 'wide' format each row is often a site/subject/patient and you have multiple observation variables. These can be either repeated observations over time, or observation of multiple variables (or a mix of both). You may find data input may be simpler or some other applications may prefer the 'wide' format. However, many of `R`'s functions have been designed assuming you have 'long' format data. This tutorial will help you efficiently transform your data regardless of original format.
+ 
+> draw this on the board? I'd like to include a diagram here too
```

Comment:
Same comment as earlier about this aside note to the instructor. Should be formalized or removed before merging.